apps/authentication/appconfig.py

```diff
@@ -18,3 +24,4 @@ def ready(self):
         watson.register(
             OnlineUser, fields=("first_name", "last_name", "ntnu_username", "nickname")
         )
+        post_save.connect(set_default_privacy, sender=User)
```

Comment:
I'm a bit unsure on how signals work, but maybe it should be `OnlineUser`, there is something about them not reacting to subclasses